v2/pkg/virtualbmc/ipmi_server.go

```diff
@@ -0,0 +1,124 @@
+package virtualbmc
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/cybozu-go/log"
+	"github.com/vishvananda/netlink"
+)
+
+// IPMIServer represents IPMI Server
+type IPMIServer struct {
+	bridges []*Bridge
```

Comment:
link netlink.Link
```suggestion
	link netlink.Link
```

Reply:
Removed network handlings from BMCServer and leave it to a client that uses BMCServer.